src/internal/motion.ts

```diff
@@ -4,4 +4,4 @@ import { findUpUntil } from './utils/dom';
 
 export const isMotionDisabled = (element: HTMLElement): boolean =>
   !!findUpUntil(element, node => node.classList.contains('awsui-motion-disabled')) ||
-  (window.matchMedia && window.matchMedia('(prefers-reduced-motion: reduce)')?.matches);
+  (!!window.matchMedia && window.matchMedia('(prefers-reduced-motion: reduce)')?.matches);
```

Comment:
Should this be `||`?

Reply:
```suggestion
!window.matchMedia || window.matchMedia('(prefers-reduced-motion: reduce)').matches;
```

Reply:
ok, another one

```suggestion
  window.matchMedia?.('(prefers-reduced-motion: reduce)').matches) ?? false;
```

Maybe also break down the function into two, for readability

```js
const isMotionDisabled = (element: HTMLElement): boolean => hasOptOutClass(element) || hasMediaQuery() ?? false;